src/cloudai/_core/installables/file.py

```
@@ -0,0 +1,72 @@
+        if self.installed_path != self.src:
+            self.installed_path.unlink()
+            self._installed_path = None
+            return InstallStatusResult(True)
+        logging.debug(f"File {self.installed_path} does not exist.")
+        return InstallStatusResult(True)
+
+    def is_installed(self, installer: "BaseInstaller") -> InstallStatusResult:
+        installed_path = installer.system.install_path / self.src.name
+        if installed_path.exists() and installed_path.read_text() == self.src.read_text():
+            self.installed_path = installed_path
+            return InstallStatusResult(True)
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)